build_all/mac/build.sh

```diff
@@ -12,7 +12,7 @@ PYTHON_VERSION=2.7
 process_args()
```

Comment:
Not checkin blocker, but I'd strongly encourage having as much of the mac's build.sh and Linux build.sh be common.  In particular the `cmake` call below in Linux build.sh is done by invoking the C SDK's build.sh which abstracts a bunch of this away.